dashboard/src/db.json

```diff
@@ -0,0 +1,1174 @@
+      "brand": "dynamode"
+    },
+    {
+      "id": "9ye984f8xqtkr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/10608119.jpg"
+      ],
+      "title": "Dynamode 3D Sound (5.1) USB-SoundCard 2.0 Gray Adapter",
+      "rating": 3.51,
+      "price": 53,
+      "category": "sound_cards",
+      "brand": "dynamode"
+    },
+    {
+      "id": "pveyhdit33mkr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/10608061.jpg"
+      ],
+      "title": "Dynamode C-Media 108 (7.1) USB-SOUND7 Adapter",
+      "rating": 3.09,
+      "price": 58,
+      "category": "sound_cards",
+      "brand": "dynamode"
+    },
+    {
+      "id": "1c45q527aaskr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/6151372.jpg"
+      ],
+      "title": "RAM Kingston DDR4-2666 8192MB PC4-21300 ValueRAM (KVR26N19S6/8)",
+      "rating": 1.34,
+      "price": 2500,
+      "category": "ram",
+      "brand": "kingston"
+    },
+    {
+      "id": "lqivo0gh5mokr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big/194516620.jpg"
+      ],
+      "title": "RAM Kingston SODIMM DDR4-2400 4096MB PC4-19200 (KCP424SS6/4)",
+      "rating": 1.89,
+      "price": 759,
+      "category": "ram",
+      "brand": "kingston"
+    },
+    {
+      "id": "vbdh6bsx26rkr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/36439229.jpg",
+        "https://content1.rozetka.com.ua/goods/images/big_tile/36439238.jpg"
+      ],
+      "title": "RAM Kingston SODIMM DDR4-2666 8192MB PC4-21300 ValueRAM (KVR26S19S6/8)",
+      "rating": 1.81,
+      "price": 2700,
+      "category": "ram",
+      "brand": "kingston"
+    },
+    {
+      "id": "ym1uizveicakr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/12593655.jpg"
+      ],
+      "title": "RAM Kingston SODIMM DDR4-2666 8192MB PC4-21300 (KVR26S19S8/8)",
+      "rating": 2.49,
+      "price": 1250,
+      "category": "ram",
+      "brand": "kingston"
+    },
+    {
+      "id": "4qh0zq6skm4kr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/11107145.jpg"
+      ],
+      "title": "RAM Kingston DDR4-2666 8192MB PC4-21300 (KVR26N19S8/8)",
+      "rating": 5,
+      "price": 1075,
+      "category": "ram",
+      "brand": "kingston"
+    },
+    {
+      "title": "123",
+      "price": "100",
+      "brand": "1",
+      "rating": "5",
+      "category": "1",
+      "id": "932b876b34176"
+    }
+  ],
+  "categories": [
+    "Monitors",
+    "Laptops",
+    "Video cards",
+    "Gaming keyboards",
+    "Computer mouse",
+    "SSD",
+    "Sound cards",
+    "RAM"
+  ],
+  "brands": [
+    "Asus",
+    "Acer",
+    "Apple",
+    "Dell",
+    "Dynamode",
+    "Gigabyte",
+    "Kingston",
+    "Lenovo",
+    "Logitech",
+    "MSI",
+    "BenQ"
+  ]
+}
```

Comment:
**Issue**: The JSON data includes hard-coded URLs and potentially sensitive data (IDs and product details) which should be managed more securely or loaded from a more secure source. <br> **Fix**: Consider loading sensitive data from environment variables or secure storage solutions to avoid exposing them in the codebase. <br> **Code Suggestion**: 
 ```
 -      "id": "bp7satk44xrkr9kjkav",
 +      "id": process.env.PRODUCT_ID_1,
 -      "images": ["https://content2.rozetka.com.ua/goods/images/big/21329737.jpg"]
 +      "images": [process.env.IMAGE_URL_1]
 ```